examples/rvc-app/rvc-common/src/rvc-device.cpp

```diff
@@ -134,6 +134,30 @@ void RvcDevice::HandleOpStateResumeCallback(Clusters::OperationalState::GenericO
                                      : to_underlying(OperationalState::ErrorStateEnum::kUnableToCompleteOperation));
 }
 
+void RvcDevice::HandleOpStateGoHomeCallback(Clusters::OperationalState::GenericOperationalError & err)
+{
+    switch (mOperationalStateInstance.GetCurrentOperationalState())
+    {
+    case to_underlying(OperationalState::OperationalStateEnum::kStopped): {
```

Comment:
Please sync this with the test plan: https://github.com/CHIP-Specifications/chip-test-plans/pull/3915/files

Reply:
This already conforms to these test plan changes.